src/main/java/bio/terra/service/dataset/flight/ingest/DatasetIngestFlight.java

```diff
@@ -133,7 +143,8 @@ public DatasetIngestFlight(FlightMap inputParameters, Object applicationContext)
 
     addStep(new IngestSetupStep(datasetService, configService, cloudPlatform));
 
-    if (ingestRequestModel.getFormat() == IngestRequestModel.FormatEnum.JSON) {
+    if (ingestRequestModel.getFormat() == IngestRequestModel.FormatEnum.JSON
+        || ingestRequestModel.getFormat() == IngestRequestModel.FormatEnum.ARRAY) {
```

Comment:
I think a `DatasetIngestFlight.NOT_CSV_INGEST` static `Set<FormatEnum>` var would be useful here. You could just check if `NOT_CSV_INGEST.contains(ingestRequestModel.getFormat())` and simplify the `if` clause.

Reply:
Or stick the logic in `IngestUtils`. Not too enamored with multi-line `if` conditions.

Reply:
Fair enough.  I'll add a method to ingest utils